adagrams/game.py

```diff
@@ -1,11 +1,105 @@
+import random
+
 def draw_letters():
-    pass
+    hand = []
+    letters_list = []
+    LETTER_POOL = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3,
+    'H': 2, 'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6,
+    'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4,
+    'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    }
+
+    #make dictionary into a list 
+    for letter, count in LETTER_POOL.items():
+        letters_list.extend(letter * count)
+
+    #make a hand of 10 letters 
+    
+    letters = []
+    temporary_letters_list = letters_list.copy()
+    for letter in range(0,10):
+        
+        random_number = random.randint(0,len(temporary_letters_list)-1)
+
+        #remove the letter from list 
+        random_letter = temporary_letters_list.pop(random_number)
+        letters.append(random_letter)
+    
+    return letters 
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
```

Comment:
Nice work on ensuring that all the letters where of the same casing. You could also use `casefold`, which is used for caseless matching. You can find more information on the method [here](https://docs.python.org/3/library/stdtypes.html#str.casefold).